docker-compose.yml

```diff
@@ -3,6 +3,7 @@ version: "3"
 services:
 
   development:
+    platform: linux/x86_64
```

Comment:
I think we would be better off instead adding a note to the readme that indicates how one can specify the platform for dev. I do not believe this line should be necessary/default for this template as specifying the platform is a bit of a workaround for mac m1/arm architecture.

This line should not be necessary in most applications.